src/Microsoft.Health.Dicom.Core/Extensions/DicomTagExtensions.cs

```diff
@@ -22,7 +23,30 @@ public static class DicomTagExtensions
         public static string GetPath(this DicomTag dicomTag)
         {
             EnsureArg.IsNotNull(dicomTag, nameof(dicomTag));
-            return dicomTag.Group.ToString("X4") + dicomTag.Element.ToString("X4");
+            return dicomTag.Group.ToString("X4").ToUpperInvariant() + dicomTag.Element.ToString("X4").ToUpperInvariant();
+        }
+
+        /// <summary>
+        /// Get default VR for dicom tag.
+        /// </summary>
+        /// <param name="dicomTag">The dicm tag</param>
+        /// <returns>The default VR.</returns>
+        public static DicomVR GetDefaultVR(this DicomTag dicomTag)
+        {
+            EnsureArg.IsNotNull(dicomTag, nameof(dicomTag));
+            if (dicomTag.IsPrivate)
+            {
+                // Private tag don't have default VR.
+                return null;
+            }
+
+            if (dicomTag.DictionaryEntry == DicomDictionary.UnknownTag)
+            {
+                // this tag is invalid
+                return null;
+            }
+
+            return dicomTag.DictionaryEntry.ValueRepresentations.FirstOrDefault();
```

Comment:
>FirstOrDefault [](start = 65, length = 14)

nit: while convenient, checking and using array access is faster and doesn't involve the extra allocation from the `IEnumerable<T>`:
```csharp
return dicomTag.DictionaryEntry.ValueRepresentations.Length > 0 ? dicomTag.DictionaryEntry.ValueRepresentation[0] : null;
 ```